lib/commands/app/create.ts

```diff
@@ -37,16 +38,22 @@ export default class AppCreateCmd extends Command {
 
 		Create a new balena application.
 
-		You can specify the application device type with the \`--type\` option.
-		Otherwise, an interactive dropdown will be shown for you to select from.
+		You can specify the organization the application should belong to using
+		the \`--organization\` option, and the application device type with the
+		\`--type\` option.
+
+		If no device type or organization is specified, interactive dropdowns
+		will be shown for you to select from.
 
 		You can see a list of supported device types with:
 
 		$ balena devices supported
-`;
+	`;
+
 	public static examples = [
 		'$ balena app create MyApp',
-		'$ balena app create MyApp --type raspberry-pi',
+		'$ balena app create MyApp --organization MyOrg',
+		'$ balena app create MyApp -o MyOrg --type raspberry-pi',
```

Comment:
Possibly related: https://github.com/balena-io/balena-ui/issues/4349

If that issue means that no 2 orgs (even from different users) can have the same fancy name, then it might be OK to identify orgs by fancy name only.

What is the acceptable character set of org fancy names? We should probably test with non ASCII characters, say an org named `οργάνωση`.

Reply:
If we were to accept org handles in addition to fancy names (which might be a nice thing for users' scripts), we'd have to have a command to list the org handles. Say, `balena orgs`.

Reply:
@pdcastro Heres
 - Org handles are unique, but mixed case.  I'll ask about the uniqueness constraints.
 - Yeah, I have org access on prod support account, had to add to staging (didn't have a team_member account there)
 - Will ask about value char sets
 - Org handles === fancy names as far as I understand.

Reply:
@srlowe @thgreasi, some potentially troubling finding re org name charset, conversion of fancy org names to handles, and this PR's design decision of identifying orgs by fancy name rather than handle:

I used the web dashboard, logged in as `gh_paulo_castro`, to create Orgs with the following fancy names:
* `ascii-org`
* `οργάνωση `

Then I used the CLI (this PR) to create apps in those orgs. Results:
```
$ ./bin/balena-dev app create test35 --type raspberrypi3 -o ascii-org
BalenaOrganizationNotFound: Organization not found: ascii-org

$ ./bin/balena-dev app create test34 --type raspberrypi3 -o οργάνωση
BalenaOrganizationNotFound: Organization not found: οργάνωση
```

Adding some log output (as suggested by Scott, thanks!), check the org handles that the web dashboard (or SDK or API?) created for those orgs:

```
[
  {
    id: 172338,
    created_at: '2020-12-10T14:00:35.505Z',
    name: 'ascii-org',
    handle: 'ascii_org',
    company_name: null,
    website: null,
    industry: null,
    billing_account_code: null,
    __metadata: { uri: '/resin/organization(@id)?@id=172338' }
  },
  {
    id: 172337,
    created_at: '2020-12-10T13:57:58.151Z',
    name: 'οργάνωση',
    handle: '_',
    company_name: null,
    website: null,
    industry: null,
    billing_account_code: null,
    __metadata: { uri: '/resin/organization(@id)?@id=172337' }
  },
```

Specially the `οργάνωση` org, which resulted in a handle of `_` ...

Reply:
Right, looks like we need to clarify this before we continue.

Reply:
Further discussion: https://www.flowdock.com/app/rulemotion/p-organizations/threads/c1PZ2OS_rNmIRI3UJqKbzv306k-

Reply:
I think discussion is resolved now?

Reply:
Yes, the resolution being that the `--organization` should specify an org handle, rather than its name.